fs/dafilesrv/dafilesrv.cpp

```diff
@@ -787,18 +799,17 @@ int main(int argc,char **argv)
             return ret;
 #endif
     }
-#ifndef _CONTAINERIZED
     {
         Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(logDir.str(), "DAFILESRV");
         lf->setCreateAliasFile(false);
         lf->setMaxDetail(TopDetail);
         lf->beginLogging();
     }
+    write_pidfile(componentName.str());
 #else
```

Comment:
May want a comment here showing this else goes with checking containerized, just because there are nested #ifdefs.

Reply:
will add a // _CONTAINERIZED here